File: Button1/app/src/main/java/com/example/button1/MainActivity.java
package com.example.button1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{
    private Button btn1,btn2;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn1 = findViewById(R.id.btn1_click);
        btn2 = findViewById(R.id.btn2_click);

        btn1.setOnClickListener(this);
        btn2.setOnClickListener(this);
    }
            @Override
                    public void onClick(View view) {
                switch (view.getId()) {
                    case R.id.btn1_click:
                        Toast.makeText(getApplicationContext(), "BTN1 DI KLIK", Toast.LENGTH_SHORT).show();
                        break;

                    case R.id.btn2_click:
                        Toast.makeText(getApplicationContext(), "BTN2 DI KLIK", Toast.LENGTH_SHORT).show();
                        break;

                }

            }
}

File: Button/app/src/main/java/com/example/button/MainActivity.kt
package android.cianjur.developer.net.basicandroid

import android.R
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    private var tombol: Button? = null // Membuat Variable Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        tombol = findViewById(R.id.testing) // Menginisialisasi ID untuk Button

        // Sebuah Listener untuk penanganan kejadian saat tombol diklik
        tombol!!.setOnClickListener {
            // Memunculkan pesan berupa text dialog saat tombol "Klik Disini" diklik.
            Toast.makeText(applicationContext, "Selamat Datang Di WILDAN TECHNO ART", Toast.LENGTH_SHORT).show()
        }
    }
}